homeassistant/components/tankerkoenig/sensor.py

```diff
@@ -0,0 +1,169 @@
+"""Platform for tankerkoenig sensor integration."""
+import logging
+
+from datetime import timedelta
+import pytankerkoenig
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_RADIUS,
+    CONF_SCAN_INTERVAL,
+    STATE_CLOSED,
+    STATE_OPEN,
+)
+from .const import NAME, CONF_STATIONS, CONF_TYPES, FUEL_TYPES
+from .base_sensor import FuelPriceSensorBase
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_RADIUS = 5
```

Comment:
From my point of view there is no "good" default for the radius. 5 km is way to large if you are in a city and might be too small if you are based on the country side. The website uses 2 km as default.

Reply:
I've changed the default to match the one from the website. Or do you think it would be better to make the radius a required configuration parameter?